### ISSUE TITLE
Are all acl clauses relative clauses?

### ISSUE THREAD

**Issue by ftyers** [ftyers/docs#3]
Here are two examples which we have considered relative clauses:
- Тез   жүгіретін   спортшылар   көрдін .
- Некенің тоқтатылуы жаңадан некеге тұру құқығын туғызады .

That is, with verbal adjectives (жүгіретін) and verbal nouns (тұру).

Here is a third example that I'm unsure of:

```
Құлдық пен құл саудасына , қандай түрде болса да , тыйым салынады . 
Slavery and slave trade , what-kind form-in may-be also , prohibition is-placed-on .

acl(түрде-7, болса-8) 
advmod(болса-8, да-9)
```

There is also the question of "optional depictives", can anyone come up with an example in Turkic of an optional depictive "She entered the room [sad]", "She painted her house [naked]" ?

**Comment by makazhan** [comment]
May be we could follow this intuition:
In constructions where verbs modify nominals:
- use `acl:relcl(V,N)` [relative] if a verb is in a past participle form V.[ҒА]н or the construction can be rephrased as a finite clause in which the modified nominal is the agent of the action (without changing the meaning);
- use `acl(V,N)` [pure] in all other cases.

Compare:

```
Сүйген қызы - A girl [who] he loved
Айтқан сөзі - a word [that] she said
[even without possessive] Барған жер - a place [smb.] went
Домбыра ойнайтын бала - Бала домбыра ойнайды [same] - a boy *who* plays dombyra
Сайлау құқығы - Құқық сайлайды [not the same] - a right *to vote*
Баратын жер - Жер барады [not the same] - a place *to go*
Өлер күн - Күн өледі [not the same] - a day to die
Келер күн - Күн келеді [same] - a day *which* comes
```

Regarding depictives, [this paper](https://www.uni-muenster.de/imperia/md/content/allgemeine_sprachwissenschaft/dozenten-unterlagen/himmelmann/himmelmann___schultze-berndt_2004_depictive_secondary_predicates_in_crosslinguistic_perspective.004.pdf) provides several examples for Turkish:

```
çay-i   tea-ACC
soguk   cold
iç-ti-k drink-PST-1PL
(We drank the tea cold) with acl(soguk,çayi) [dependent first]
```

A Kazakh version would be:

```
шайды   tea-ACC
суықтай cold-ADV [derivation]
іштік   drink-PST-1PL
```

_шайды суық іштік_ may also be possible but I couldn't find such examples
The problem is that object can be referenced to a preceding sentence leaving an orphaned node, e.g.

```
Шай жылытатын     тоқ     болған жоқ.
Tea   warm up electricity   was  not
  Амалсыздан    суықтай іштік.
Without options  cold   drank
```

In the latter case the only option seems to be `advmod(суықтай,іштік)`

Similarly:

```
Ол   бөлмеге мұңлы   күйде   кірді
She room.DAT  sad  state.LOC entered
amod(мұңлы,күйде) and acl(күйде,Ол)
vs.
 бөлмеге мұңлы   күйде   кірді
room.DAT  sad  state.LOC entered
amod(мұңлы,күйде) and nmod:npmod(күйде,кірді)
```

Lastly, regarding Fran's example of `acl(түрде-7, болса-8)`, assuming that the order is (gov, dep), isn't that `nmod:npmod(болса-8, түрде-7)` or `iobj(болса-8, түрде-7)`, or even `cop(түрде-7, болса-8)`?

**Comment by jonorthwash** [comment]
If I understand, you seem to be assuming that a relative clause is one where the subject is "extracted" from the verb phrase (/predicate) and used as some element in another phrase.  My understanding is that a relative clause is one where any nominal argument is "extracted" from the verb phrase (/predicate) and used as some element in another phrase.  This definite encompasses your examples like сайлау құқығы, баратын жер, etc.  In this sense it's used roughly the same as `acl`.

Maybe we need to pin down a consistent definition of "relative clause" before we continue forward, and it should probably be one we can point people to later.

**Comment by ftyers** [comment]
I made a mistake in my original example (that will teach me to write issues when really tired!)

```
Құлдық пен құл саудасына , қандай түрде болса да , тыйым салынады . 
Slavery and slave trade , what-kind form-in may-be also , prohibition is-placed-on .

acl(саудасына-4, түрде-7) 
cop(түрде-7, болса-8) 
advmod(болса-8, да-9)
```

(Order: gov, dep)

Sorry about that!

**Comment by makazhan** [comment]
Jonathan, maybe my choice of examples implied subject extraction, but in general, I agree that it's any nominal argument.
My intuition was that in order for an `acl` clause (C) that modifies a nominal (N) to be considered relative, its host sentence/clause (H) should be expressible as two sentences that both contain N. In the first sentence N will be whatever it was in H, and in the second one it would be an argument of the VP/predicate (in finite form) of C:

Домбыра ойнайтын баланы көрдім [`obj(баланы,көрдім)`] =
= Баланы көрдім + Бала домбыра ойнайды [`subj(бала,ойнайды)`]
Бала appears in both and it's an argument of _ойнайды_, so we have  `acl:relcl(ойнайтын,баланы)`
vs
Оның сүйген қызын көрдім [`obj(қызын,көрдім)`] =
= Қызын көрдім + Ол қызды сүйді [`obj(қызды,сүйді)`]
Conditions are satisfied, so we have  `acl:relcl(сүйген,қызды)`
vs
Сайлау құқығын алдым (I received a right to vote) [`obj(құқығын алдым)`] = 
= құқығын алдым + ???(құқығын,сайлау)
The second condition is not satisfied (we can say _құқық сайлады_ or _құықыты сайладым_, but it will not relate to the first sentence), so we have _pure_ `acl(құқыға,сайлау)`

**Comment by jonorthwash** [comment]
Okay, I understand now what you're saying.  I'm open to the possibility that there are `acl` relations in Kazakh that shouldn't be considered relative clauses.

However, I'm not sure about your example.  Here сайлау is a verbal noun (in indefinite genitive), not a verbal adjective like the other examples.  I share your intuition that these are different from the other examples you gave.  On the other hand, I think the difference might not be exactly what you think it is.  Couldn't you reword it to "Құқық алдым.  Құқығы(м) - сайлау"?

**Comment by makazhan** [comment]
Indeed it can. And the difference may not be what I think. Following my logic any(?) `acl` can be broken into two sentences, as your "Құқық алдым. Құқығым - сайлау" decomposition shows. Even some English `acl` can:
_[points to establish are X](http://universaldependencies.github.io/docs/en/dep/all.html#al-en-dep/acl:relcl)_ = points are X + these points are to establish (to be established)
_you have a right to remain silent_ = you have a right + your right is to remain silent

I think the finitness of a clause should be stressed here. I mentioned it in my previous post but didn't put enough emphasis. UD docs state that relative clauses are finite. So, I have this slightly stretched intuition, but bear with me.

tmk, in Kazakh, other than а/е/й aorist and [дт][ыі] past, there are tense suffixes that have exact same surface forms as participle suffixes do. Let's compare on бар-TENSE-2Sg example:

```
ертең бар-а-сың | tomorrow go-FUT-2Sg  >>> ертең бар-ар-сың | ертең бар-мақ-сың
кеше бар-ды-ң | yesterday go-PST-2Sg >>> кеше бар-ған-сың | кеше бар-ып-сың
жиі бар-а-сың | often go-AOR-2Sg >>> жиі бар-атын-сың | жиі бар-а-*тұғын*-сың
```

I'm not saying that _домбыра ойнаған_ in _домбыра ойнаған бала_ is a finite clause (with a null-surface 3Sg/Pl agreement), but at least it has a capacity of being finite, e.g. бала домбыра ойнаған (a boy [used to play] played dombyra).
Anyway the point is that while verbal adjectives can in fact be finite verbs in certain contexts, у/ю-ending verbal nouns cannot. The only way I can see them used as a finite predicate is substanivization as in "Құқығым - сайлау".

So, on the basis of finite-incapability can we treat verbal nouns as `"pure acl"` and all verbal adjective instances and other finite clauses (as in _шашы ұзын қыз_) as `acl:relcl`?

**Comment by jonorthwash** [comment]
> I'm not saying that _домбыра ойнаған_ in _домбыра ойнаған бала_ is a finite clause (with a null-surface 3Sg/Pl agreement), but at least it has a capacity of being finite, e.g. бала домбыра ойнаған (a boy [used to play] played dombyra).

Yes, I'd say this is right.

> tense suffixes that have exact same surface forms as participle suffixes do

You can say "сайлау_да_мын", though maybe this doesn't count as the finite counterpart of this gerund.  What about the verbal adjective suffix -`GIс` (can you say "сайлағыс адам"??) or the verbal adjective/noun (?) -`ушI` ("сайлаушылар", "сайлаушы адам")?  These can't really be used as finite forms (any more than -`у` can be).  Whatever the case may be, using a morphological argument for dependency tagging doesn't satisfy me—arguments should be based more on the relations between the words, whatever their morphology is like.

> The only way I can see them used as a finite predicate is substanivization as in "Құқығым - сайлау".

Or subjects, right?  E.g., "Саулау - азаматтың міндеті."  Or as an object "Республика сайлауды қашан өткізеді?"  Or any number of other things.  And actually, I'm seeing stuff online like "ҚРдағы сайлау" instead of "ҚРда сайлау", which starts to suggest this particular word being lexicalised (i.e., a _deverbal_ noun).  Could you say "бассейндегі жүзу" (as a noun phrase in some sentence)?  But I guess you could say "Жүзу құқығы", or better yet "қалаған кандидатқа сайлау құқығы" so it doesn't matter—this construction is definitely gerundy.

> So, on the basis of finite-incapability can we treat verbal nouns as `"pure acl"` and all verbal adjective instances and other finite clauses (as in _шашы ұзын қыз_) as `acl:relcl`?

I do share your intuition that this is different from other `acl`s.  Let's start by talking about UD's definition of a relative clause: "characterized by finiteness and usually omission of the modified noun in the embedded clause".  It's weird that the only _solid_ criterion that they give is a morphological one.  I much prefer [Wikipedia's definition](https://en.wikipedia.org/wiki/Relative_clause): "a kind of subordinate clause that contains an element whose interpretation is provided by an antecedent on which the subordinate clause is grammatically dependent".  So by UD's criteria, there are no relative clauses in Kazakh (since none of the forms are finite), but by Wikipedia's definition (which is a lot more in line with my intuition), these are definitely relative clauses.  I believe examples like "сайлау құқығы" (or "бассейнде жүзу құқығы" etc.) also fit this definition.

Now, **even if this sort of construction shouldn't be considered a relative clause**, I would say that since it is much less common than verbal_adjective-based relative clauses, it should have the "special" (or longer / more complex) tag.  I.e., I would argue for verbal_adjective-based `acl`s always just being labeled `acl` (unless there is something else special about them), and these as maybe `acl:ccomp` or something.

A quick question that I don't think has any bearing on my thoughts here, but might just help clarify what's going on: can you say simply "бассейнде жүзу құқық", without possession morphology?

**Comment by makazhan** [comment]
> A quick question that I don't think has any bearing on my thoughts here, but might just help clarify what's going on: can you say simply "бассейнде жүзу құқық", without possession morphology?

As a native speaker, I have never heard nor seen such a usage. To be more or less certain, I checked hand-annotated data at my disposal: out of 7315 `(verbal noun).NOM + noun` pairs 208 happened to lack possession morphology on the trailing noun. Some of those cases were not clausal noun modifiers, e.g.  _...некелесу (getting married) орын алуда (happening)_. In `acl` cases the trailing nouns were just parts of bigger phrases whose governors were in possessive form, e.g.  _...жабдықтау (equipping) су айналымЫның (of circulation)_.

> Let's start by talking about UD's definition of a relative clause (RC): "characterized by finiteness and usually omission of the modified noun in the embedded clause". It's weird that the only solid criterion that they give is a morphological one.

If by a morphological criterion you mean finiteness, I agree, it is strange.

> by Wikipedia's definition (which is a lot more in line with my intuition), these are definitely relative clauses.

Assuming that "these" refer to verbal nouns that modify nominals and that we agree that verbal adjectives modifying nominals are all RCs, we are back to square one.

== THE acl/RC ISSUE ==

As the title of the issue indicates, this discussion started from my intuition that in Kazakh all clauses that modify nominals are essentially RCs. This is what I wrote in our Turklang paper: 

> We did not specify the clausal noun modifier relation (acl), because, as much as we tried, we could not find examples where such clauses were not relative (acl:relcl).

However, I am hesitant to treat all clausal noun modifiers identically for the following reasons:
1) There's definitely something about verbal nouns that sets them apart from other clausal noun modifiers;
2) There's the issue of choosing a single label/relation:
-- If we choose to use `acl` for everything that would mean that there are no relative clauses in Kazakh, because `acl` is used strictly for modifiers that are _not_ RCs;
-- If I am not mistaking, UD does not allow using X:Y relations without defining X. That is we cannot use `acl:relcl` for everything without having at least one case for `acl`. (Something which I proposed in our Turklang paper and which I now strongly oppose);
-- If we use `acl` for verbal adjectives and smth. like `acl:ccomp` for verbal nouns (as Jonathan proposed), we again leave out `acl:recl` (special RC relation). From a cross-UD consistency point of view this doesn't look good to me;
-- If we compromise: use `acl:recl` for verbal adj-s and smth. like `acl:ccomp` for verbal nn-s, we, again, face "X:Y without X" UD-style issue.

== POSSIBLE SOLUTION ==
I propose that we make the following design decision (which I outlined in my previous post):
- Use clausal noun modifier relation, `acl` for verbal nouns, e.g. _сайлау құқығы_.
- Use relative clause relation, `acl:relcl` for verbal adjectives (_ойнаған бала_) and predicative clauses (_шашы ұзын қыз_ → `acl:relcl(қыз, ұзын)`)

So, even if verbal nn-s are, indeed, RCs, I propose to make a compromise and call them `acl`. This way we can label RCs (at least obvious ones) with appropriate relation, and have `acl` to be consistent and to satisfy UD-style requirement of having X for each X:Y relation.

== OPTIONAL DEPICTIVES ==
As for optional depictives, i.e. _She entered the room sad_ → `acl(she, sad)`; _We drank the tea cold_, etc. These are usually adverbalized in Kazakh either morphologically (suffix -ДАй) or analytically (with the help of _бол-_ and _күй-_): Шайды суық_тай_ іштік → `advmod/advcl(іштік,суықтай)`; Бөлмеге мұңлы _болып/күйде_ кірді → `advmod/advcl(кірді,мұңлы болып/күйде)`.

**Comment by ftyers** [comment]
I like the possible solution. Also, this is something that can be at least partly determined morphologically. If it's tagged as a verbal adjective, then it's going to be `acl:relcl`. The verbal nouns will of course require inspection, but that's what we'd expect anyway. For the optional depictives, I'm happy with either `advmod` or `advcl`, depending on the morphology.

**Comment by jonorthwash** [comment]
I can live with the proposal.  I'd just like most of what you said above to be included in the documentation, especially the fact that these may well be relative clauses from a linguistic point of view.

For the record, this relation is a lot like the N-N.px compound-type relation we've been labeling `nmod`, except it's with a verb form.  After we've made a lot of these decisions, we should look into the structures where verb forms are behaving much the same as other parts of speech and be explicit about the equivalencies.  E.g., we already have things like `advcl` ≈ `advmod`, `acl:relcl` (now) ≈ `amod`, etc.  In my opinion, in Turkic, there's not much difference in syntax (function and behaviour) between the tags in these pairs—they're only different because of the morphology / different parts of speech are involved (which in turn means you can have some stuff subsumed in them, which lets things like relative clauses happen).

**Comment by jonorthwash** [comment]
As for optional depictives, I suggest we just analyse those normally.  They don't really appear to be a special syntactic phenomenon in Kazakh the way they are in English.
`advmod(іштік, суықтай)`
and
`advcl(кірді, мұңлы)`
`cop(болып, мұңлы)`
as well as
`nmod(кірді, күйде)`
`amod(мұңлы, күйде)`

As this is all fairly productive, I don't think we should be lexicalising things like "мұңлы болып" and "мұңлы күйде".

**Comment by makazhan** [comment]
@jonorthwash 

> For the record, this relation is a lot like the N-N.px compound-type relation we've been labeling nmod, except it's with a verb form.

It is indeed.

> After we've made a lot of these decisions, we should look into the structures where verb forms are behaving much the same as other parts of speech and be explicit about the equivalencies.

Here's another interesting one: Kazakh -ҒЫр optatives(?). Consider these verses:
_Олай жүрші, Көк Шолақ!
Былай жүрші, Көк Шолақ!
Жылдамдат_**қыр** _Көк Шолақ,
Арам қат_**қыр** _Көк Шолақ._
(c) [Кене Әзірбаев - Көк Шолақ](https://youtu.be/qeFabUGwFhQ?t=1m39s)
_Көк Шолақ_ is the name of the singer's horse, _lit._ Gray Short Tailed

Here:
Жылдам-дат-қыр Көк Шолақ = "[may you] speed up" Көк Шолақ
Арам қат-қыр Көк Шолақ = "may you drop dead" Көк Шолақ

These are interesting because they have optative modality and seem to be primarily attributive, behaving like [verbal] adjectives:
-- agree: арам қатқыр-сың
-- substativize: арам қатқыр-лар, арам қатқыр-дың...
For comparison, -ҒАй optatives agree (_[есіңе мені ал-ғай-сың](https://youtu.be/tHDr7qAg_yI?t=5m20s)_), but, tmk, do not substantivize.

Although _арам_.Adj _қат_\- is phrasal, analytical constructions are also used, e.g. _[Атың өшкір Годзилла](http://blogtime.kz/cinema/atyn-oshkir-godzilla.html)_ (_"May your name disappear" Godzilla_).

Given their verbal adjective-like behavior, I would analyze -ҒЫр optatives as `acl:relcl`:

```
1 Атың        2 nsubj
2 өшкір       3 acl:relcl
3 Годзилланы  4 dobj
4 көрдім      0 root
```

What do you guys think about them?

A potential problem is the fact that some of -ҒЫр forms seem to be completely deverbalized adjectives, e.g. _біл-гір_ (knowledgeable), _өт-кір_ (sharp, compare: _өтпейтін пышақ_), etc. If there is not enough context it may proove difficult to separate `amod` from `acl:relcl`...

P.S. Maybe these forms are indeed verbal adjectives, but I know of at least one "pure" optative usage (heard it a lot, but can't find written usage): _Жаның шықсын, жаның шыққыр!_. It is a bit tautological: _Let your soul leave you, you "may your soul leave"_

**Comment by jonorthwash** [comment]
I would say that -KIр is a verbal adjective suffix, and it would seem like it can work like optative too.  I would say that optative (and imperative and other similar forms, including -KAй) is probably a sort of finite form since it would be considered `root` (and many agree in various persons, but this isn't a good criterion).

Incidentally, Kyrgyz has a verbal adjective -KIс which is like the negative form of this one, e.g. алгыс хат, бүткүс азап, ичкис суу, etc. (meaning roughly алмайтын, бітпейтін, ішпейтін).
